components/dashboard/src/projects/NewProject.tsx

```diff
@@ -0,0 +1,314 @@
+                        <label htmlFor="selectAccount" className="font-medium">Account</label>
+                        <select name="selectAccount" value={selectedAccount} className="w-full"
+                            onChange={(e) => onSelectAccount(e.target.value)}>
+                            {accounts.map(a => (<option key={`account-${a}`} value={a}>{`${a} (${provider})`}</option>))}
+                            {isGitHub() && (<option value="addOrg">Add GH Org</option>)}
+                            <option value="selectProvider">Select Git Provider</option>
+                        </select>
+                    </div>
+                    <div className="flex-col mt-4">
+                        {reposToRender.length > 0 && (
+                            <div className="overscroll-contain max-h-96 overflow-y-auto">
+                                {reposToRender.map(r => (
+                                    <div key={`repo-${r.name}`} className="rounded-md whitespace-nowrap flex space-x-2 py-2 px-3 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light transition ease-in-out group">
+                                        <div className="w-8/12 text-gray-400 flex flex-col">
+                                            <span className="text-gray-800 dark:text-gray-300 text-base font-semibold overflow-ellipsis truncate">{toSimpleName(r.name)}</span>
+                                            <span className="">Updated __ ago</span>
+                                        </div>
+                                        <div className="w-4/12 flex justify-end">
+                                            <div className="flex self-center hover:bg-gray-200 dark:hover:bg-gray-700 rounded-md cursor-pointer opacity-0 group-hover:opacity-100">
+                                                <button className="primary py-1" onClick={() => selectRepo(r)}>Select</button>
+                                            </div>
+                                        </div>
+                                    </div>
+                                ))}
+                            </div>
+                        )}
+                    </div>
+                </div>
+                <div className="p-3 bg-gray-100">
+                    <div className="text-gray-500 text-center">
+                        Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
+                    </div>
+                    {noOrgs && (
+                        <div className="text-gray-500 mx-auto text-center">
+                            Missing organizations? <a href="javascript:void(0)" onClick={e => grantReadOrgPermissions()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Grant permissions</a>
```

Comment:
**issue**: This is not applicable when switching to a GitLab provider, right?

Reply:
At the moment this is GitHub specific. 
TODO: disable for GitLab